cbits/energy-rapl.c

```diff
@@ -1,28 +1,40 @@
 #include <unistd.h>
+#include <pthread.h>
 #include "rapl.h"
 
 static int fd_msr = 0;
 static struct rapl_units r_units;
 static int units_ok;
 
+static double energy_sum = 0;
+pthread_t thread;
+
+void *thread_sum_energy_packet(void *arg)
+{
+    for (;;)
+    {
+        sleep(15);
+        struct rapl_raw_power_counters rpc;
+        rapl_get_raw_power_counters(fd_msr, &r_units, &rpc);
```

Comment:
Awesome. I was reading [perf's source](https://github.com/torvalds/linux/blob/master/arch/x86/kernel/cpu/perf_event_intel_rapl.c#L170. Just leaving here for future reference.